testplan/exporters/testing/pdf/renderers/base.py

```diff
@@ -138,8 +138,8 @@ def get_metadata_context(self, source):
         """
         return collections.OrderedDict(
             [
-                (label, source.meta[key])
+                (label, dict(source.information)[key])
                 for key, label in self.get_metadata_labels()
-                if source.meta.get(key)
+                if dict(source.information).get(key)
```

Comment:
use key in ... instead of the get, it is nicer to read unless you want to ignore all keys that does not exist and the ones which are falsy